test/extended/builds/cluster_config.go

```diff
@@ -45,12 +138,14 @@ var _ = g.Describe("[Feature:Builds][Serial][Slow][Disruptive] alter builds via
 				exutil.DumpPodLogsStartingWith("", oc)
 				exutil.DumpConfigMapStates(oc)
 			}
-			oc.AsAdmin().Run("apply").Args("-f", defaultConfigFixture).Execute()
-			oc.AsAdmin().Run("apply").Args("-f", defaultbuildConfigFixture).Execute()
```

Comment:
Note this approach in fact did *NOT* reset the `cluster` build.config instance in my testing.  I left `defaultConfigFixture` alone since the image related stuff is being skipped, and the card for this PR if focused on build.